server/models/Collective.js

```diff
@@ -746,7 +765,7 @@ export default function(Sequelize, DataTypes) {
     }
 
     if (this.type === 'USER') {
-      if (user && user.email && this.name && this.name !== 'anonymous') {
+      if (user && user.email && this.name && this.name !== 'incognito') {
```

Comment:
We wan't to keep that for previous "anonymous" accounts.

```suggestion
      if (user && user.email && this.name && this.name !== 'anonymous' && this.name !== 'incognito') {
```